src/components/input-number/input-number.tsx

```diff
@@ -258,6 +258,14 @@ export class InputNumber
    */
   @Prop({ reflect: true }) step?: number | "any";
 
+  /**
+   * Specifies the type of content to autocomplete, for use in forms.
+   * Read the native attribute's documentation on MDN for more info.
+   *
+   * @mdn [step](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)
+   */
+  @Prop({ reflect: true }) autocomplete?: "string";
```

Comment:
@benelan shouldn't it be `autocomplete: string;`?

We don't need the optional question mark and the type should be string not "string" for these right?

Reply:
Do these need to be reflected?

Reply:
Woops my bad, fixing. I copy and pasted from the `step` property which has "any" in quotes for some reason. I'll fix that in a separate PR